### PR TITLE
Contributors

### PR DESCRIPTION
**Pull request by alocke12992**

### CODE REVIEW

File: website/static/js/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            <div class='contributor'>
+              <a href='${item.html_url}' target='_blank' ref='noopener noreferrer'>
+                <h2 class='contributorLogin'>
+                  ${item.login}
+                </h2>
+              </a>
+              <img class='contributorImg' src=${item.avatar_url} alt='${item.login} avatar' />
+            </div>  
+            `
+          )).join('')}
```
Comment (frehner) [inline website/static/js/index.js:20]: want to have a `.catch` here just in case?

File: website/pages/en/contributors.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const React = require('react')
+const CompLibrary = require('../../core/CompLibrary.js'); 
+const Container = CompLibrary.Container;
+const GridBlock = CompLibrary.GridBlock;
```
Comment (kentmclean) [inline website/pages/en/contributors.js]: Unused item?

File: website/static/js/index.js (hunk 1 above)
Comment (alocke12992) [reply website/static/js/index.js:20]: smart. Thanks for the .catch ;)

File: website/pages/en/contributors.js (hunk 2 above)
Comment (alocke12992) [reply website/pages/en/contributors.js]: Yup, good catch.

**Pull request merged**